pkg/controller/networkpolicy/store/networkpolicy.go

```diff
@@ -165,6 +167,16 @@ func NewNetworkPolicyStore() storage.Interface {
 			}
 			return groupNames, nil
 		},
+		PerNamespaceRuleIndex: func(obj interface{}) ([]string, error) {
+			fp, ok := obj.(*types.NetworkPolicy)
+			if !ok {
+				return []string{}, nil
+			}
+			if len(fp.PerNamespaceSelectors) > 0 {
+				return []string{HasPerNamespaceRule}, nil
```

Comment:
Use true?

Reply:
index func needs `[]string` as return type, so I don't see much value in returning true here?

Reply:
Done.